src/sortStudents.ts

```diff
@@ -1,16 +1,61 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' |'desc';
 
+function getAverageGrade(grades: number[]): number {
```

Comment:
you can use destructuring, so when you call the function you can just pass an object and it will automaticly extract grades
```suggestion
function getAverageGrade({ grades }: Student): number {
```